### PR TITLE
[SDK, REACT] Switch to simpler detection API for ERC20/721/1155 extensions

### PR DESCRIPTION
**Pull request by joaquim-verges**

### CODE REVIEW

File: packages/sdk/src/constants/erc20-features.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const FEATURE_TOKEN = {
   features: {
     [FEATURE_TOKEN_BURNABLE.name]: FEATURE_TOKEN_BURNABLE,
     [FEATURE_TOKEN_MINTABLE.name]: FEATURE_TOKEN_MINTABLE,
-    [FEATURE_TOKEN_DROPPABLE.name]: FEATURE_TOKEN_DROPPABLE,
+    [FEATURE_TOKEN_ClAIMABLE.name]: FEATURE_TOKEN_ClAIMABLE,
```
Comment (jarrodwatts) [inline packages/sdk/src/constants/erc20-features.ts:76]: Lowercase "L"

Comment (jnsdls) [reply packages/sdk/src/constants/erc20-features.ts]: nice find

**Pull request merged**